bin/release.py

```diff
@@ -129,11 +130,19 @@ def get_remote_name(github_user):
 
 def make_tag(bug_number, remote_name, tag_name, commits_since_tag):
     """Tags a release."""
-    message = "\n".join(commits_since_tag)
-
     if bug_number:
-        # Add bug number to tag
-        message = message + f"\n\nDeploy bug #{bug_number}"
+        resp = fetch(BZ_BUG_JSON_URL + bug_number, is_json=True)
+        bug_summary = resp["bugs"][0]["summary"]
+
+        input(f">>> Using bug {bug_number}: {bug_summary}. Correct? Ctrl-c to cancel")
```

Comment:
When I read a similar message while deploying this morning I admit I paused for a second and hit enter, not 100% sure what I was supposed to do in case I *didn't* want to cancel.
Doesn't prevent me from approving, though

Reply:
I'm game for different prompting. If you have ideas on what would feel more intuitive (or anything else), write up an issue in the socorro-release issue tracker.